### PR TITLE
Add tests for Part URI helper

### PR DESCRIPTION
**Pull request by twsouthwick**

### CODE REVIEW

File: test/DocumentFormat.OpenXml.Packaging.Tests/PartUriHelperTests.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        {
+            var helper = new PartUriHelper();
+            var unique = helper.GetUniquePartUri(contentType, new Uri(parentUri, UriKind.Relative), new Uri(targetUri, UriKind.Relative));
+
+            Assert.Equal(new Uri(expected, UriKind.Relative), unique);
+        }
+
+        [InlineData("unknown", "/", "/target", "/target2")]
+        [InlineData("unknown", "/something", "/target", "/target2")]
+        [InlineData("application/vnd.openxmlformats-officedocument.wordprocessingml.footer+xml", "/target", "target", "/target1")]
```
Comment (twsouthwick) [inline test/DocumentFormat.OpenXml.Packaging.Tests/PartUriHelperTests.cs]: @Josh-Curry You added support for some namespaces to append the 1, but it appears that this is only the case for one of the overloads of `PartUriHelper.GetUniqueParUri`. Is this expected, or should it be added for all?

**Comment by twsouthwick** [comment]
@tomjebo @tarunchopra Can either of you weigh in here?

**Comment by tomjebo** [comment]
@twsouthwick I'll take a look shortly

**Comment by twsouthwick** [comment]
@tomjebo @Will-Gregg I'll plan on merging this in EOD unless I hear otherwise

**Comment by twsouthwick** [comment]
Thanks. The CI system builds and run tests and verifies no app compat issues :)

**Pull request merged**